scripts/exglobal_land_analysis_prep.sh

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env bash
```

Comment:
We prefer all new exscripts added to be in python unless there is a very strong argument to not do so.

Reply:
As mentioned above, this work can be done after moving the `apply_jedi_increments` into the `gdaslandanlfinal` job.